plugins/modules/system/puppet.py

```diff
@@ -54,10 +54,10 @@
   logdest:
     description:
     - Where the puppet logs should go, if puppet apply is being used.
-    - C(all) will go to both C(stdout) and C(syslog).
+    - C(all) will go to both C(console) and C(syslog).
     type: str
-    choices: [ all, stdout, syslog ]
-    default: stdout
+    choices: [ all, console, syslog ]
+    default: console
```

Comment:
You definitely have to keep the `stdout` option to avoid breaking backwards compatibility. I wonder whether the name changed over time, and originally `stdout` **was** the correct value (in which case we can't simply replace it).

Reply:
I have not specified it in docs, but it will be set to `console` if `stdout` has been used. Do you want to me add it to docs again, this would result in more users choosing wrong option IMO.

Reply:
You also have to keep it in the docs. It's probably be best to deprecate the old value by code (and mention that in the docs), this will more effectively prevent people from using it. (Also if someone is used it and it is the correct value for their version of puppet, they are more likely to complain so we find out ;) )

Reply:
FYI I asked this in `#puppet` IRC channel on Freenode and someone told me it's been `console` since `puppet` 2.7 at least:

from Puppet 2.7, puppet agent -help: 
```
 --logdest:
  Where to send messages. Choose between syslog, the console, and a log
  file. Defaults to sending messages to syslog, or the console if
  debugging or verbosity is enabled.
```

Reply:
Considering that Puppet 2.7.0 is from 2011, and the option for this module was 'only' added in 2016, it should be safe to change the value. I'm wondering a bit why Puppet didn't complain about the wrong value...